### ISSUE TITLE
Possible mistake in alpha loss in SAC

### ISSUE THREAD

**Issue by perrin-isir** [DLR-RM/stable-baselines3#802]
Hello, 
In https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/sac/sac.py, line 215:
```python
ent_coef_loss = -(self.log_ent_coef * (log_prob + self.target_entropy).detach()).mean()
```
The leading coefficient is `self.log_ent_coef`, but in equation (18) in https://arxiv.org/pdf/1812.05905.pdf, it is alpha that is present in the loss, not log(alpha).  
Similarly, in the implementation of the authors (https://github.com/rail-berkeley/softlearning/blob/master/softlearning/algorithms/sac.py
, line 256): 
```python
alpha_losses = -1.0 * (self._alpha * tf.stop_gradient(log_pis + self._target_entropy))
```
So shouldn't line 215 of sac.py be modified accordingly?

The same potential mistake is present in the Tianshou implementation (https://github.com/thu-ml/tianshou/blob/master/tianshou/policy/modelfree/sac.py, line 177), so I wonder: is there actually a reason for this difference? In any case, I guess that it has a very limited impact on the algorithm.

**Comment by perrin-isir** [comment]
Sorry I didn't see https://github.com/rail-berkeley/softlearning/issues/37

**Comment by araffin** [comment]
Duplicate of  #712 and #36

**Comment by perrin-isir** [comment]
Thanks!
